### PR TITLE
Limits logging after round

### PR DESCRIPTION
**Pull request by Archanial**
## About The Pull Request

Logging stuff such as:
- mech logs
- virus logs
- cloning logs
- access logs
- attacks logs
- bomb logs
- mob specific attack logs (the ones seen in LOGS panel in game)
- investigate logs

won't be logged after the round ends.

Say logs, admin logs and rest are kept and unchanged.

## Why It's Good For The Game

Helps admins investigate stuff and reduces bloat of logs caused by BR.

## Changelog
:cl:
admin: Reduced logging after the round.
/:cl:

### CODE REVIEW

**Comment by PowerfulBacon** [comment]
> id logs / manifest 

Log this for N bombs

**Comment by Archanial** [comment]
> 
> 
> > id logs / manifest
> 
> Log this for N bombs

I totally wish we didn't need to log that but you are right

**Comment by PowerfulBacon** [comment]
Maybe it would be better to go with a GLOB.round_state and add the round state to the logs so you can clearly see which points are at:
 - Init
 - Pregame
 - Game
 - Post game

**Comment by Archanial** [comment]
The whole idea was to reduce logging not add additional logging stating current state of the game.

I think logs from Init and Pregame are pretty distinguishable, while Game and Post Game are similar.

File: code/__HELPERS/_logging.dm [hunk 1]
```diff
@@ -89,22 +89,25 @@
 	WRITE_LOG(GLOB.world_asset_log, "ASSET: [text]")
 
 /proc/log_access(text)
-	if (CONFIG_GET(flag/log_access))
+	if (CONFIG_GET(flag/log_access) && SSticker.current_state != GAME_STATE_FINISHED)
```
Comment (bloons3) [inline code/__HELPERS/_logging.dm]: Log Access we DEFINITELY want to continue logging even after round end

**Comment by Rukofamicom** [comment]
This is something that would be great for log diving, especially when it comes to major shuttle incidents.

**Review by utilbird — APPROVED** [review]

**Pull request merged**